mirai-core-api/src/commonMain/kotlin/contact/AvatarSpec.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019-2021 Mamoe Technologies and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
+ *
+ * https://github.com/mamoe/mirai/blob/dev/LICENSE
+ */
+
+package net.mamoe.mirai.contact
+
+/**
+ * 头像的规格, [size] 单位 px.
+ */
+@Suppress("unused")
+public enum class AvatarSpec(public val size: Int) : Comparable<AvatarSpec> {
```

Comment:
```suggestion
public enum class AvatarSpec(@MiraiInternalApi public val size: Int) : Comparable<AvatarSpec> {
```

注意还要添加 import

如果没有特别需求, 一般尽快少地提供 public API, 减轻维护负担